src/silverlabnwb/nwb_file.py

```diff
@@ -583,15 +591,24 @@ def add_stimulus(self):
             self.nwb_file.add_stimulus(TimeSeries(**attrs))
         self._write()
 
-    def read_cycle_relative_times(self, file_path):
+    def read_cycle_relative_times_pre2018(self, file_path):
         """Read the 'Single cycle relative times.txt' file and store the values in memory.
 
         Note that while the file has times in microseconds, we convert to seconds for consistency
         with other timestamps in NWB.
         """
         assert os.path.isfile(file_path)
-        self.cycle_relative_times = pd.read_csv(file_path, names=('RelativeTime', 'CycleTime'),
-                                                sep='\t', dtype=np.float64) / 1e6
+        timings = LabViewTimingsPre2018(file_path)
+        self.cycle_relative_times = timings.pixel_time_offsets
+        self.cycle_time = timings.cycle_time
+
+    def read_cycle_relative_times_v231(self, file_path, roi_path):
+        timings = LabViewTimings231(file_path,
+                                    roi_path=roi_path,
+                                    n_cycles_per_trial=self.imaging_info.cycles_per_trial,
+                                    n_trials=len(self.trial_times))
+        self.cycle_relative_times = timings.pixel_time_offsets
+        self.cycle_time = timings.cycle_time
```

Comment:
Since these are quite similar, it may be better to have a single method `read_cycle_relative_times` which takes a path rather than a file name:
```python
def read_cycle_relative_times(self, path):
    file_path = ...
    if version is old:
        timings = LabViewTimingsPre2018(file_path)
    else if version is new:
        roi_path = ...
        timings = LabViewTimings231(file_path, roi_path)
    self.cycle_relative_times = ...
```

Or we could even have the Timings classes take a path in their constructor, but that makes them less general (and makes testing harder).

Reply:
We should also update the docstring to replace the specific filename with something more descriptive that applies to both versions.